apps/client/src/pages/Dashboard/Settings/index.tsx

```diff
@@ -0,0 +1,142 @@
+import classNames from 'classnames'
+import { Text } from 'components'
+import { useState } from 'react'
+import styles from './settings.module.scss'
+const data = [
+  {
+    section: 'Communication Settings',
+    items: [
+      {
+        label: 'Email Notifications',
+        description: 'Receive Important Updates and Notifications via email',
+        accessor: true,
+      },
+    ],
+  },
```

Comment:
let's add **disabled** for each item...

I see a possibility where backend can disable and option for a user

Reply:
oh i didnt know you dropped a comment here oo boss, once you review this i will add it